Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.10
+ENV PYTHONDONTWRITEBYTECODE 1
```

Comment:
I _think_ this might worsen the performance? Keen to get your thoughts @matthew-shaw

Reply:
...I don't think this is too important actually. It might be better to have it in to avoid writing `.pyc` files.